code/modules/mob/living/carbon/carbon.dm

```diff
@@ -598,4 +598,36 @@ var/const/GALOSHES_DONT_HELP = 4
 		if(locate(/obj/item/device/assembly/health) in src)
 			stat(null, "Health: [health]")
 
-	add_abilities_to_panel()
\ No newline at end of file
+	add_abilities_to_panel()
+
+/mob/living/carbon/proc/add_screams(var/list/screams)
+	if(!alternate_screams)
```

Comment:
That should never happen unless you manually set it to null,so iunno how needed it is